code/modules/multiz/basic_legacy.dm

```diff
@@ -2,18 +2,15 @@
 GLOBAL_LIST_EMPTY(connected_z_cache)
 
 // todo: legacy code
-/proc/GetConnectedZlevels(z)
-	return SSmapping.get_z_stack(z)
```

Comment:
This is not equivalent to what it was changed to, comms goes down regularly and RCON does not work between Z-levels anymore